### DISCUSSION TITLE
About avalonia web programming

### DISCUSSION THREAD

**Discussion by zy1075984** [AvaloniaUI/Avalonia#11935] [Q&A] [answered]
More and more people are interested in avalonia web programming, can you explain how avalonia uses signalR to communicate with the server? Thank you!

**Comment by timunie** [comment, marked as answer]
maybe a good topic fot the docs. Let me transfer this ticket over to it.

**Reply by timunie** [reply]
~~See: https://github.com/AvaloniaUI/avalonia-docs/issues/66~~

**Reply by timunie** [reply]
See also: https://github.com/AvaloniaUI/Avalonia/discussions/11921
